cmd/erasure-healing.go

```diff
@@ -861,8 +861,12 @@ func isAllBucketsNotFound(errs []error) bool {
 	}
 	notFoundCount := 0
 	for _, err := range errs {
-		if err != nil && errors.Is(err, errVolumeNotFound) {
-			notFoundCount++
+		if err != nil {
+			if errors.Is(err, errVolumeNotFound) {
```

Comment:
Can we merge them as `if errors.Is(err, errVolumeNotFound) || isErrBucketNotFound(err)` ?

Reply:
hmm we can